pwnlib/util/proc.py

```diff
@@ -94,7 +94,7 @@ def match(p):
 
     processes = sorted(processes, key=lambda p: p.create_time())
```

Comment:
`reverse=True` should be here, then you can skip the `reversed` call down there.

https://docs.python.org/2/library/functions.html#sorted
https://docs.python.org/3/library/functions.html#sorted

```suggestion
    processes = sorted(processes, key=lambda p: p.create_time(), reverse=True)
```